tsfile/src/test/java/org/apache/iotdb/tsfile/read/ReadOnlyTsFileTest.java

```diff
@@ -139,5 +140,19 @@ void queryTest2() throws IOException {
     }
     Assert.assertEquals(10647, cnt);
   }
+
+  void queryNonExistPathTest() {
+    ArrayList<Path> paths = new ArrayList<>();
+    paths.add(new Path("dr.s1"));
+    paths.add(new Path("d2.s1"));
+    IExpression expression = new GlobalTimeExpression(TimeFilter.gt(1480562664760L));
+    QueryExpression queryExpression = QueryExpression.create(paths, expression);
+    try {
+      QueryDataSet queryDataSet = tsFile.query(queryExpression);
+    } catch (IOException e) {
+      Assert.assertNotNull(e.getMessage(), "No measurement path : dr");
```

Comment:
Add a `fail()`  after `QueryDataSet queryDataSet = tsFile.query(queryExpression);` in case that the Exception does not rise.
Why use `assertNotNull`?